packages/mobile/src/dappkit/dappkit.ts

```diff
@@ -70,7 +69,7 @@ function* produceTxSignature(action: RequestTxSignatureAction) {
       // In walletKit we use web3.eth.getCoinbase() to get gateway fee recipient
       // but that's throwing errors here. Not sure why, but txs work without it.
       const gatewayFeeRecipient = undefined
-      const gatewayFee = '0x' + new BigNumber(10000).toString(16)
+      const gatewayFee = undefined
```

Comment:
Here we make the fee undefined but in walletkit you use `0x0`. Which is the recommended approach?

Reply:
I set it as `0x0` instead of `undefined` undefined there because the gateway fee recipient wanted one being defined to mean the other is also defined. (It doesn't matter top geth though because it will treat `nil` as 0)